modules/DSS-05_branches.yml

```diff
@@ -6,10 +6,12 @@ screens:
       title: Using Branches
       image: github-branches.png
       presenter-script: |
-        Every day you will be editing files on your project or adding new files. You don't want to take risks with the code on your master branch so you need to create a branch.
-        
+        Branching is an essential part of using Git and GitHub. While other Version Control Systems can make branching really painful, in Git it's easy and necessary.
```

Comment:
Change to: 
"Branches are an essential concept in Git and GitHub. While other version control systems can make branching painful, branches in Git are nothing more than a lightweight way to track the most recent commit in a line of work."